cupy/random/generator.py

```diff
@@ -198,6 +198,21 @@ def normal(self, loc=0.0, scale=1.0, size=None, dtype=float):
             func = curand.generateNormalDouble
         return self._generate_normal(func, size, dtype, loc, scale)
 
+    def pareto(self, a, size=None, dtype=float):
+        """Returns an array of samples drawn from the pareto II distribution.
+
+        .. seealso::
+            :func:`cupy.random.pareto_kernel` for full documentation,
+            :meth:`numpy.random.RandomState.pareto`
+        """
+        a = cupy.asarray(a)
+        if size is None:
+            size = a.shape
+        y = cupy.empty(shape=size, dtype=dtype)
+        _kernels.pareto_kernel(a, self.rk_seed, y)
+        self.rk_seed += numpy.prod(size)
```

Comment:
Similarly to #1420, this distribution could be generated from the result of `self._random_sample_raw`.